pod/perlvar.pod

```diff
@@ -203,13 +203,13 @@ this variable, doing so is generally discouraged, although it can be
 invaluable for some testing purposes.  It will be reset automatically
 across C<fork()> calls.
 
-Note for Linux and Debian GNU/kFreeBSD users: Before Perl v5.16.0 perl
+Note for Linux and Debian GNU/kFreeBSD users: Before Perl v5.16.0 Perl
 would emulate POSIX semantics on Linux systems using LinuxThreads, a
 partial implementation of POSIX Threads that has since been superseded
 by the Native POSIX Thread Library (NPTL).
 
 LinuxThreads is now obsolete on Linux, and caching C<getpid()>
-like this made embedding perl unnecessarily complex (since you'd have
```

Comment:
AFAICT this is literally talking about embedding the perl interpreter, so this instance should not be capitalized.